lsp-dls.el

```diff
@@ -0,0 +1,56 @@
+;;; lsp-dls.el --- dls configuration               -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2020 Hiroki Noda
```

Comment:
Is it better to use `lsp-mode developers` ?

Reply:
It is fine to have it like that.